requirements.txt

```diff
@@ -3,4 +3,5 @@ linux-metrics==0.1.4
 pint==0.7.2
 paho-mqtt==1.2
 aenum==1.4.5
-CoAPthon==4.0.2
\ No newline at end of file
+CoAPthon==4.0.2
+uptime==3.0.1
```

Comment:
hello @winniex1 

This is a new addition to Liota and will require approval from legal team. The process has already been completed please check with the Product team on how to get the approval for this new library.